docs/end-user-guide/using-the-plugin.md

```diff
@@ -0,0 +1,13 @@
+## Connecting an account
+
+Use the `/mstmeetings connect` slash command to connect.
+
+## Starting a call
+
+Start a call either by clicking the header button or the `/mstmeetings start` slash command. Every meeting you create will create a new meeting room in MS Teams. If two meetings are created really close (less than 30 seconds) you will get a prompt asking you whether you are sure you want to create the meeting.
+
+## Disconnecting an account
+
+You can disconnect your MS Teams account from Mattermost using the `/mstmeetings disconnect` slash command.
+
+To connect another account to Mattermost, follow the configuration instructions again. If you are disconnecting in order to refresh the token, then use `/mstmeetings connect` to reconnect.
```

Comment:
I think this line is not needed.

Reply:
The whole paragraph or just the last line?

Reply:
The whole paragraph